Complete Ledger

A5rocks opened the pull request:
This should work... At least, it works on my machine:tm:

n0remac commented:
This is totally incompatible with the code I wrote for the ledger and less complete. I don't see any graphics and /graphics/generated appears to be empty.

A5rocks committed:
Let the ledger display cuneiform *correctly*.

I ran into two problems while doing this:
  1) `kivy.uix.image.Image` only takes strings as the source, I was using `pathlib.Path`
  2) Every `kivy.uix.image.Image` was displayed on a seperate line.

The first I fixed by using `str` and `.resolve()`.
The second I fixed by using more `kivy.uix.boxlayout.Boxlayout`s!

It's Boxlayouts all the way down!

A5rocks commented:
This comes with a generator.

Edit `graphics/cuneiform_one.png` and `graphics/cuneiform_ten.png` as you want, run `pipenv run generate` and it generated based on what you supplied.

I thought that was what we wanted?
